single-line-diagram/single-line-diagram-core/src/main/java/com/powsybl/sld/svg/styles/BusesHighlightStyleProviderFactory.java

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+package com.powsybl.sld.svg.styles;
+
+import com.powsybl.iidm.network.Network;
+import com.powsybl.sld.svg.SvgParameters;
+import com.powsybl.sld.svg.styles.iidm.HighlightLineStateStyleProvider;
+import com.powsybl.sld.svg.styles.iidm.TopologicalStyleProvider;
+
+public class BusesHighlightStyleProviderFactory implements StyleProviderFactory {
```

Comment:
```suggestion
public class BusHighlightStyleProviderFactory implements StyleProviderFactory {
```
I tend not to use plural when nouns are used to qualify something but I am not 100% sure.